bootcamp/serializers_test.py

```diff
@@ -0,0 +1,36 @@
+"""Tests for general bootcamp serializing functionality"""
+from unittest.mock import patch, Mock
+from django.test import TestCase
```

Comment:
I am not say that you should rewrite these tests now, but didn't we say that new modules should be `pytest` style?